packages/renderer/src/lib/recommendation/ExtensionBanner.svelte

```
@@ -33,10 +33,10 @@ async function onClose(): Promise<void> {
   let result: MessageBoxReturnValue = { response: -1 };
   try {
     result = await window.showMessageBox({
-      title: 'Hide extension recommendation banners',
+      title: 'Hide Extension Recommendations?',
       message: `Do you want to hide extension recommendation banners?`,
```

Biome (2.4.11)

[error] Do not use template literals if interpolation and special-character handling are not needed.

(lint/style/noUnusedTemplateLiteral)